lib/default-config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = () => {
   const config = {
     baseURL: 'https://selenium-release.storage.googleapis.com',
-    version: '3.141.59',
+    version: '4.0.0-alpha-3',
```

Comment:
Please revert this change

Reply:
This one still should be reverted

Reply:
Ping

Reply:
sorry I was still testing this using the config. I'll revert this to the original once the tests pass. (i know it's messy)